locales/en.default.schema.json

```diff
@@ -51,7 +51,7 @@
         },
         "type_body_font": {
           "label": "Font",
-          "info": "Selecting a different font can affect the speed of your store. [Learn more about system fonts.](https://help.shopify.com/en/manual/online-store/os/store-speed/improving-speed#fonts)"
+          "info": "Selecting different fonts can affect the speed of your online store. Learn more about [system fonts.](https://help.shopify.com/en/manual/online-store/os/store-speed/improving-speed#fonts)"
```

Comment:
Sorry, I was hoping this could become a 1x support copy vs repeating under each Header and Body fonts but that would have to be later if even possible.

Would cancel these revisions for now.